winrt/winrt.d.ts

```diff
@@ -10171,6 +10171,7 @@ declare module Windows {
             getFilesAsync(): Windows.Foundation.IAsyncOperation<Windows.Foundation.Collections.IVectorView<Windows.Storage.StorageFile>>;
             getFoldersAsync(): Windows.Foundation.IAsyncOperation<Windows.Foundation.Collections.IVectorView<Windows.Storage.StorageFolder>>;
             getItemsAsync(): Windows.Foundation.IAsyncOperation<Windows.Foundation.Collections.IVectorView<Windows.Storage.IStorageItem>>;
```

Comment:
Not related to this PR but this signatue
```ts
 getItemsAsync(): Windows.Foundation.IAsyncOperation<Windows.Foundation.Collections.IVectorView<Windows.Storage.IStorageItem>>;
```
seems *wrong* : https://msdn.microsoft.com/en-us/library/windows/apps/br227287.aspx?cs-save-lang=1&cs-lang=javascript#code-snippet-1